package/yast2-services-manager.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Thu Jan 19 17:10:47 CET 2017 - schubi@suse.de
+
+- Using systemctl calls (is-active, is-enabled) to evaluate if the
+  service is enabled/active (bnc#1017166, 1012047)
```

Comment:
I think you should use the `bnc#` prefix so when the package is updated we get the notification in the bug report.